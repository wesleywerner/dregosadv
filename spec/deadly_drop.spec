superbrief
gonear cliff portal
cross the bridge
cross the bridge
cross the bridge
cross the bridge
cross the bridge
cross the bridge
use whip
wait
undo
whip the bridge
