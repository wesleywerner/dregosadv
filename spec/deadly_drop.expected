  THIS GAME IS IN DEBUG MODE
  
  Dr Ego and the Egg of Man-Toomba
  An Interactive Fiction Story
  by Special Agent.
  Release 1 / Serial number 400410 / Inform v6.33 Library v6.12.3pre SX
  
  The Maro River (in the canoe)
. 
  In the canoe you can also see a trap (which is closed and empty), a fishing
  spear and a fishing net.
  
> >  Dr Ego and the Egg of Man-Toomba is now in its "superbrief" mode, which
  always gives short descriptions of locations (even if you haven't been
  there before).
  
> >  Gorge
  Some stone stairs leads downward. A rope bridge is due east.
  
> >  Waterfall
  A temple is due south. A rope bridge is due west.
  
> >  Gorge
  Some stone stairs leads downward. A rope bridge is due east.
  
> >  Waterfall
  A temple is due south. A rope bridge is due west.
  
> >  Gorge
  Some stone stairs leads downward. A rope bridge is due east.
  
> >  Waterfall
  A temple is due south. A rope bridge is due west.
  
> >  Deadly Drop
  
  You step on a rotten plank and it cracks under your weight, you lose your
  balance!
  
> >  Start your sentence with WHIP, like WHIP THE TREE.
  Teetering the other way, your fedora tumbles into the gorge below -- You
  are about to fall!
  
> >  Time passes.
  
  You plummet into the gorge below, cracking your head on a rocky
  outcropping, far down below. Weeks pass before the small search party
  discover your broken body. Luckily you did not suffer.
  
  (If only you tried your WHIP)
  
      *** You have died ***
. 
  In that game you scored 0 out of a possible 108, in 8 turns.
  
  Would you like to RESTART, RESTORE a saved game, UNDO your last move, give
  the FULL score for that game or QUIT?
> >  Deadly Drop
  [Previous turn undone.]
  
> >  You swing your whip and *SNAP* it on the rope bridge. Dangling
  precariously, you pull yourself back up to safety. That was close!
  
  Waterfall
  A temple is due south. A rope bridge is due west.
  
> >